nixos/services/matomo.nix

```diff
@@ -98,6 +98,228 @@ let
     '';
   };
 
+  # This script was generated by copying recommended commands from the Matomo web ui
```

Comment:
I'd like to see this script as a separate file as it is quite long and hard to read inline.